### DISCUSSION TITLE
possible to ID locations of words, not just chars?

### DISCUSSION THREAD

**Discussion by wdchild** [jsvine/pdfplumber#916] [Q&A]
I've spent some time exploring `pdfplumber`, including taking a look at the `page.chars` and `page.rects` objects and  the `page.extract_text` method. Chars come with their positions well documented in the returned data. But is there a sensible way to identify the positions / boundary boxes of individual words in the text? For scanned images, engines like tesseract allow you to bound individual words (or what it perceives as words). I'm seeing no easy way to do that with plumber. What am I missing? Thanks!

**Comment by jsvine** [comment]
Hi @wdchild, I think you're looking for the `page.extract_words(...)` method. You can learn more about what the method does and the parameters it accepts in the ["Extracting text" section of this project's README.md file](https://github.com/jsvine/pdfplumber#extracting-text). Does that help?
